drivers/temperature/max31865pmb1/max31865.c

```diff
@@ -190,13 +191,13 @@ int max31865_write(struct max31865_dev *device, uint8_t reg_addr,
 		.cs_change = 1,
 	};
 
-	if(reg_addr < MAX31865_CONFIG_REG || reg_addr >= MAX31865_FAULTSTAT_REG)
+	if (reg_addr < MAX31865_CONFIG_REG || reg_addr >= MAX31865_FAULTSTAT_REG)
 		return -ENXIO;
 
-	if(reg_addr == MAX31865_RTDMSB_REG || reg_addr == MAX31865_RTDLSB_REG)
+	if (reg_addr == MAX31865_RTDMSB_REG || reg_addr == MAX31865_RTDLSB_REG)
 		return -ENXIO;
 
-	if((reg_addr & MAX31865_WRITE_MASK) != MAX31865_WRITE_MASK)
+	if ((reg_addr & MAX31865_WRITE_MASK) != MAX31865_WRITE_MASK)
```

Comment:
Not entirely related to this change, but can't this `if(...)` and just do the or operation anyway?

Reply:
Will remove the if statement and just perform the or operation anyway since the first if(...) statement already checks if reg_addr is valid.